### ISSUE TITLE
Spreadsheet Export to Excel does not work properly when the validation type is 'list'

### ISSUE THREAD

**Issue by nelito987** [telerik/kendo-ui-core#4346]
### Bug report
When there is a cell with validation type 'list' and the Spreadsheet is exported to Excel, there is an alert when trying to open the file.

### Reproduction of the problem
1. Open the[ Dojo example](https://dojo.telerik.com/@NeliKondova/iLIxoMuW)
2. Click the 'Export' button
3. Open the exported excel file

### Current behavior
Message "We found problem with some content in 'Workbook.xlsx'. Do you want us to try to recover as much as we can?... " appears. Such message does not appear with the other validation types such as 'date'.

### Expected/desired behavior
The behavior should be consistent. There should be no alerts when opening a file exported from Spreadsheet.

Last working version: 2016.3.118

### Environment

* **Kendo UI version:** 2018.2.620
* **Browser:** [all ]

**Comment by mishoo** [comment]
Actually I just found out that Excel does not support an array of strings for the validation — it must be either a reference (so items are fetched from the sheet) or a single string with comma-separated values.  Therefore, instead of:

```
from: '{ "Foo item 1", "Bar item 2", "Baz item 3" }'
```

we should use:

```
from: '"Foo item 1,Bar item 2,Baz item 3"'
```

I pushed a fix to allow this.  Exporting the document with this kind of validation will then work in Excel.
